docker/Dockerfile

```diff
@@ -7,18 +7,20 @@ COPY ./go-tf-prepare/main.go main.go
 COPY ./go-tf-prepare/pkg/ pkg/
 RUN GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o tf-prepare main.go
 
-FROM alpine:3.12.1
+FROM debian:bullseye-slim
 
-RUN mkdir -p /tmp/install /usr/src /work
-WORKDIR /tmp/install
+#Base
+RUN apt update -y && \
+    apt install -y git curl openssl && \
+    apt install -y pip && \
+    apt install -y gcc libffi-dev libssl-dev python3-dev make unzip gpg wget && \
```

Comment:
I think we can remove python3-dev, it's only needed when compiling c libs if I remember correctly and I don't think yq needs it. Give it a try .
I'm also unsure how much we need libffi-dev libssl-dev

We will need make since the pipeline builds on it and if I remember correctly gcc is needed to install make. @bittrance probably have some input on this one. But you can try to remove it as well and see if it installs it any way.